### PR TITLE
Add github folder and templates to project

### LINKED ISSUE #4: Create template directory
You know what to do

### PR DESCRIPTION
**Pull request by JoshSevy**
# Crate Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

- Add .github folder with templates for issues and pull requests

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other, Please explain: **Project Setup NO CODE**

## Has This Been Tested?
- [x] No
- [ ] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:

# Message for reviewer:

Here is a good start for the templates for the projects, but we can make these our own or modify to better fit us as a team. Thanks all excited to work with everyone on this project.

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+### Feature Categories:
+
+
+- [ ] Express
+- [ ] React
+- [ ] Styling/CSS
+- [ ] Documentation
+- [ ] Other: 
+
+**Describe the feature/Bug your gonna work on**
```
Comment (rwilliams659) [inline .github/ISSUE_TEMPLATE.md:20]: ** you're 😄 
Beyond that, looks great. We should discuss what kinds of labels we want to use!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Crate Pull Request Form
+
+## Description
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+`Your message here`
+## Type of change
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Other, Please explain:
```
Comment (rwilliams659) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: I really like this particular template! When I've used it before the only thing that came up is that sometimes for "Type of change" we also needed a "Testing" category, though not sure we will for this project.

**Review by rwilliams659 — APPROVED** [review]
Thanks for adding these, Josh!

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (Abdeboskey) [reply .github/ISSUE_TEMPLATE.md:20]: In addition, (although not directly related) we should norm on Branch naming conventions too!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (Abdeboskey) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: I remember them mentioning today that we will need to do testing, so I think that's an excellent suggestion. Yeah, the level of detail/coverage here is crate.

**Review by Abdeboskey — APPROVED** [review]
Really nice to just have these setup from the get-go. Thanks Josh!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (Strompy) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: I like using the checkboxes, but just as a heads up GH considers them part of a list so you will see something like `6/10` on PR requests

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+
+
+**Additional context**
+Add any other context or screenshots about the feature.
```
Comment (Strompy) [inline .github/ISSUE_TEMPLATE.md:25]: Screenshot are great and GH lets you just drag and drop images into the issue

**Review by Strompy — APPROVED** [review]
Love it. Thanks for the templates!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (sienna-kopf) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: Upvoting this for a testing option!

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (sienna-kopf) [reply .github/ISSUE_TEMPLATE.md:25]: No way ! That is SO cool. The things you learn on group projects.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (sienna-kopf) [reply .github/ISSUE_TEMPLATE.md:20]: You mean you don't want me to name all my branches some jumble of 
'sienna-fix-the-thing' hahaha. For real though, down on this!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (JoshSevy) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: I love adding Testing to the templates. These are our templates now. Dream Team!

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (mlynch5187) [reply .github/ISSUE_TEMPLATE.md:20]: Love the format!

**Review by mlynch5187 — APPROVED** [review]

**Comment by Abdeboskey** [comment]
Hi friends, I know we only have 4 approvals but I see comments from everyone, so I'm gonna go ahead and merge this PR.

**Pull request merged**